core/domain/summary_services.py

```diff
@@ -29,9 +31,99 @@
 from core.domain import rights_manager
 from core.domain import search_services
 from core.domain import stats_services
+from core.domain import user_domain
 from core.domain import user_services
 
-_LIBRARY_INDEX_GROUPS = [{
+from typing import Callable, Dict, List, Optional, Sequence, Tuple, Union
+from typing_extensions import TypedDict
+
+
+class DisplayableCollectionSummaryDict(TypedDict):
+    """Dict representing the displayable collection summary dictionary."""
```

Comment:
```suggestion
    """Type for the displayable collection summary dictionary."""
```

diet elsewhere where we represent a dict directly and not dict of some domain object.

Reply:
Comment is changed wherever it is applicable!